src/BlazorUI/Bit.BlazorUI/Components/Inputs/Slider/BitSlider.scss

```
@@ -15,10 +15,27 @@ $BgHover: $clr-pri-light;
         background: none;
 
         &:focus {
-            outline: none;
+            outline: none; // The visible focus indicator lives on the thumb pseudo-element below.
         }
     }
     /**/
```

Stylelint (17.11.1)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)